cores/nRF5/Print.cpp

```diff
@@ -187,11 +188,16 @@ size_t Print::println(const Printable& x)
   return n;
 }
 
+extern "C" void streamOutput(char character, void* arg)
+{
+  (static_cast<Print *>(arg))->write(character);
+}
+
```

Comment:
Can we move this outside the class definition?

Reply:
Where?

Reply:
Moved it to the top of the file.